clang/test/Sema/flex-array-extensions.c

```diff
@@ -0,0 +1,43 @@
+// RUN: %clang_cc1 %s -verify=c -fsyntax-only -fflex-array-extensions
+
+// The test checks that flexible array members do not emit warnings when
+// -fflex-array-extensions when used in a union or alone in a structure.
+
+struct already_hidden {
+	int a;
```

Comment:
Please run `git clang-format HEAD~` on  your commit. This looks like kernel coding style and not llvm coding style (which is used even for unit tests).

Reply:
I ran the helper script a few times, actually. It never flagged this, just syntax in SemaDecl.cpp

Reply:
```sh
$ cat clang/test/.clang-format
DisableFormat: true
```
:(